.github/gh_runner_dependencies.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
```

Comment:
Hey @NimJay 

Ah yeah. So the steps are as below:
- Create a new VM in the `anthos-gke-samples-ci` project
- Run this script on that VM
- Then follow the instructions on [runners page](https://github.com/GoogleCloudPlatform/anthos-samples/settings/actions/runners) when you click `Add runner` to register that VM with Github
- Use the labels given to the runners in the workflows files at `<repo_root>/.github/workflows`

I have already followed this steps to add 2 runners for this repo. In future if someone was to add a new runner they would follow these steps

Reply:
Gotcha, thanks, Shabir!
I am going to approve this PR since I can see both new VMs/runners being successfully used in some pull-request Actions](https://github.com/GoogleCloudPlatform/anthos-samples/runs/2992660132?check_suite_focus=true).
It would be nice to include the explanation you provided in your previous comment at the top of the `gh_runner_dependencies.sh` file (for any passerby), but it's not necessary.